### DISCUSSION TITLE
GenericFailure

### DISCUSSION THREAD

**Discussion by kaykhancheckpoint** [prisma/prisma#12487] [Q&A]
I have a long running script where for each i need to fetch some data from the database. However after a while i get this `GenericFailure` and i'm not really too sure what is happening here..

input - 
```
[ '29115583286', '40773967' ], 2022-02-01T00:00:00.000Z, 2022-02-28T00:00:00.000Z
```

```
export const GetReachForServiceIds = async (serviceIds: string[], startDate: Date, endDate: Date) => {
    try {
        const result = await PrismaService.reach_growth.findMany({
            where: { service_id: { in: serviceIds }, date: { gte: startDate, lte: endDate } },
        });

        return [result, null] as const;
    } catch (err) {
        Logger.error({ kind: "leaderboard_reach", method: "GetReachForServiceIds", error: err, stack_trace: err.stack });
        return [null, err] as const;
    }
};
```

> Invalid `PrismaService.reach_growth.findMany()` invocation in
> /home/kay/checkpoint/leaderboards/src/reach/reach.service.ts:19:57
> 
>   16 
>   17 export const GetReachForServiceIds = async (serviceIds: string[], startDate: Date, endDate: Date) => {
>   18     try {
> → 19         const result = await PrismaService.reach_growth.findMany(
>   
>     at Object.request (/home/kay/checkpoint/leaderboards/node_modules/@prisma/client/runtime/index.js:39818:15)
>     at async PrismaClient._request (/home/kay/checkpoint/leaderboards/node_modules/@prisma/client/runtime/index.js:40646:18)
>     at async GetReachForServiceIds (/home/kay/checkpoint/leaderboards/src/reach/reach.service.ts:19:24)
>     at async LeaderboardReach (/home/kay/checkpoint/leaderboards/src/reach/leaderboard-reach.ts:23:37)
>     at async Main (/home/kay/checkpoint/leaderboards/src/reach/main.ts:9:9) {
>   code: 'GenericFailure',
>   clientVersion: '3.11.0',
>   meta: undefined

**Comment by austincrim** [comment]
Hey @kaykhancheckpoint 👋 ,

Could you re-run your process with [debug logging](https://www.prisma.io/docs/concepts/components/prisma-client/debugging) turned on?

**Reply by kaykhancheckpoint** [reply]
> Hey @kaykhancheckpoint wave ,
> 
> Could you re-run your process with [debug logging](https://www.prisma.io/docs/concepts/components/prisma-client/debugging) turned on?

Hi, i believed this issue was to do with some bad data in my table, once i seeded the new version of the table i no longer got this error.

**Comment by kaykhancheckpoint** [comment]
Hi, i believed this issue was to do with some bad data in my table, once i seeded the new version of the table i no longer got this error.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
